src/quizzes/chapter-1/1.3.1/unix-quiz.js

```diff
@@ -0,0 +1,47 @@
+const rawQuizdown = `
+---
+shuffleAnswers: true
+---
+
+# If performed on a nonexistent file, all of the following commands will return a non-zero exit code except:
+
+1. [x] touch
+1. [ ] cat
+1. [ ] mv
+1. [ ] cp
+1. [ ] rm
+
+# If you run the following command on a nonexistent directory, what will happen?
+\`\`\`bash
+cd nonexistent-directory-name
+\`\`\`
+
+1. [x] The command will error out and do nothing
+1. [ ] The directory will be created
+1. [ ] The directory will be created AND you will navigate into it
```

Comment:
We already have a `cd` question maybe we add one around `mkdir` something like

```
Assume you are a new user and the directory /home/user/docs does not exist. Which of the following mkdir commands will successfully create the nested directories /home/user/docs/work? Check all that apply:

[] mkdir /home/user/docs/work
[x] mkdir -p /home/user/docs/work
[x] mkdir --parents /home/user/docs/work
[x] mkdir /home/user/docs && mkdir /home/user/docs/work

Explanation:

mkdir /home/user/docs/work: This command will fail because the intermediate directory (/home/user/docs) does not exist.

mkdir -p /home/user/docs/work: The -p or --parents flag allows mkdir to create parent directories as needed. This command will create all necessary intermediate directories, including docs.

mkdir --parents /home/user/docs/work: This is equivalent to the previous command, using the long-form option --parents instead of -p.

mkdir /home/user/docs && mkdir /home/user/docs/work: This command sequence first attempts to create the docs directory and then creates the work directory inside it. Since docs does not exist, it will be created successfully, followed by the successful creation of work.
```

Reply:
Kept the original question, but added this one as well 👍